lib/mustachio/rack_attack.rb

```diff
@@ -0,0 +1,25 @@
+require 'active_support/cache'
+require 'active_support/cache/memory_store'
+
+class Rack::Attack
+  RPM = 10
+
+  self.cache.store = ActiveSupport::Cache::MemoryStore.new
```

Comment:
Could move to a shared cache down the road, but this is good enough for now, and doesn't require any additional setup.